handlers/base.go

```diff
@@ -189,3 +190,12 @@ func DecodePossibleBase64(original string) string {
 
 	return decoded
 }
+
+// EncodeBase64 encodes the list of strings with Linux base64
+func EncodeBase64(strList []string) string {
```

Comment:
I might be wrong but isn't this method just concatenating strings? I know the json marshaler will use base64 for byte[] but I think we should do an explicit conversion.